sbpack/noncwl/utils.py

```diff
@@ -133,7 +165,8 @@ def zip_and_push_to_sb(api, workflow_path, project_id, folder_name):
     else:
         folder_id = folder_found[0].id
 
-    print(f'Uploading file {zip_path}, please wait for the upload to complete.')
+    print(f'Uploading file {zip_path}, '
+          f'please wait for the upload to complete.')
     u = api.files.upload(zip_path, parent=folder_id, overwrite=False)
```

Comment:
Shouldn't this use `overwrite=True`?

If changes are made to the application that was already published, the code package should be re-uploaded.

Reply:
Files will already have different names in most cases, since the name is generated at the time of file upload, and it includes the timestamp. In cases where the name of the zip is the same, I think we should not overwrite files, since that would break the app which was using the old file.